bigbluebutton/greenlight
# Branding image as room logo

## Description

Added 1 line of code to add the greenlight branding image url (when configured) as logo to the room api create call

## Testing Steps

Configure a branding image in greenlight
Create a new room
Join the room
See the greenlight branding image in the top left corner of the BBB room

## Screenshots (if appropriate):

![Screenshot from 2021-10-12 11-47-41](https://user-images.githubusercontent.com/23553596/136934735-37c87cc4-2fb8-44f9-95e9-efa5e51a1069.png)

![Screenshot from 2021-10-12 11-48-17](https://user-images.githubusercontent.com/23553596/136934751-f7c945db-9478-4c57-be95-013c06785f18.png)
